rspamd.yaml

```
@@ -54,15 +56,26 @@ pipeline:
 
   - uses: cmake/install
 
-  - uses: strip
-
   - runs: |
-      # Create required directories with proper permissions
-      install -d -m 755 "${{targets.destdir}}"/etc/rspamd
+      mkdir -p "${{targets.destdir}}"/etc/rspamd
+
+      # Create symlink to main rspamd.conf in /etc/rspamd/
+      ln -sf "${{targets.destdir}}"/usr/etc/rspamd/rspamd.conf "${{targets.destdir}}"/etc/rspamd/rspamd.conf
```

```suggestion
      ln -sf "/usr/etc/rspamd/rspamd.conf" "${{targets.destdir}}/etc/rspamd/rspamd.conf"
```

Maybe lets add a symlink check because of this? `- uses: test/tw/symlink-check` (with allow absolute.)

---

so tried the symlink approach, the current symlink is pointing to the build directory path, but at runtime, the target should be `/usr/etc/rspamd/rspamd.conf`, where as the suggested approach would failed during build because `/usr/etc/rspamd/rspamd.conf` doesn't exist yet at build time, it's only available after the package is installed

i think maybe we could copy the config directly during build where its expected, but correct me if im wrong or if you have any other suggestions:
```
cp "${{targets.destdir}}"/usr/etc/rspamd/rspamd.conf "${{targets.destdir}}"/etc/rspamd/
```